src/develop/ui/patterns/mobile/masterdetail.md

```diff
@@ -3,33 +3,44 @@ tags: runtime-mobileandreactiveweb;
 summary: 
 ---
 
-# MasterDetail Pattern
+# Master Detail
 
-The MasterDetail pattern is similar to the SplitScreen pattern, but it receives an item list for the left panel. You can use this pattern in tablet apps, to display the detail of a list of items.
+You can use the Master Detail Pattern to display a master list and the details for the currently selected item.
 
-Here's the preview in Service Studio:
+![](images/masterdetail-preview.png)
 
-![](images/Pasted_image_at_2017_05_10_04_53_PM.png)
 
-## How to Use the MasterDetail Pattern
+## How to use the Master Detail Pattern
 
-Start by binding a List to the **ItemList** parameter and leverage the Block events to change the content placeholder.
 
-1\. Create a local boolean variable and set it on **OpenedOnPhone**.
+1. In Service Studio, in the Toolbox, search for `Master Detail`. 
 
-![](images/MasterDetail_list_open.png)
+    The Master Detail widget is displayed.
+    
+    ![](images/masterdetail-widget.png)
+
+
+1. From the Toolbox, drag the Master Detail widget onto your application's screen.
+
+1. Bind a List to the **ItemList** parameter and leverage the block events to change the content placeholder.
```

Comment:
What list? And what does it mean to "leverage Block Events to change the content placeholder"?